articles/introduction-to-random-forest-in-ml/index.md

```diff
@@ -0,0 +1,99 @@
+
+A rain forest system relies on various decision trees. Every decision tree consists of a root node, decision nodes, and leaf nodes. The leaf node of each tree is the final output produced by that specific decision tree. The selection of the final output follows the majority-voting system. In this case, the output chosen by the majority of the decision trees becomes the final output of the rain forest system. The diagram below shows a simple random forest classifier. 
+
+![Random Forest Classifier](/engineering-education/introduction-to-random-forest-in-machine-learning/random-forest-classifier.png)
+
+[Image Source: Medium](https://miro.medium.com/max/5752/1*5dq_1hnqkboZTcKFfwbO9A.png)
+
+Let's take an example of a training dataset consisting of various fruits such as bananas, apples, pineapples, and mangoes. The random forest classifier divides this dataset into subsets. These subsets are given to every decision tree in the random forest system. each decision tree produces its specific output. For example, the prediction for trees 1 and 2 is *apple*. Another decision tree (n) has predicted *banana* as the outcome. The random forest classifier collects the majority voting to provide the final prediction. The majority of the decision trees have chosen *apple* as their prediction. This makes the classifier to choose *apple* as the final prediction. 
+
+![Example of Random Forest Classifier](/engineering-education/introduction-to-random-forest-in-machine-learning/example-of-random-forest-classifier.png)
+
+[Image Source: Javatpoint](https://static.javatpoint.com/tutorial/machine-learning/images/random-forest-algorithm2.png)
+
+### Applications of Random Forest
+Some of the applications of the random forest include:
+
+#### Banking
+Random forest is used in banking to predict the creditworthiness of a loan applicant. This helps the lending institution to make a good decision on whether to give the customer the loan or not. Banks also use the random forest algorithm to detect fraudsters. 
+
+#### Health care
+Health professionals use random forest systems to diagnose patients. Patients are diagnosed by assessing their previous medical history. Past medical records are reviewed to establish the right dosage for the patients. 
+
+#### Stock Market
+Financial analysts use it to identify potential markets for stocks. It also enables them to identify the behavior of stocks. 
+
+#### E-commerce
+Through rain forest algorithms, e-commerce vendors can predict the preference of customers based on past consumption behavior. 
+
+### Advantages of Random Forest
+* It can perform both regression and classification tasks.
+* A random forest produces good predictions that can be understood easily. 
+* It can handle large datasets efficiently. 
+* The random forest algorithm provides a higher level of accuracy in predicting outcomes than the decision tree algorithm.
+
+### Disadvantages of Random Forest

```

Comment:
You mention applications of random forest. Can you give specific examples where random forest would not be suited. The reason is, the examples given are all classification problems. How does one know when to use it, and when not to?